main.go

```diff
@@ -518,23 +527,34 @@ func main() {
 	cleaner := clean.New(db)
 	alertingRules := prometheus.NewAlertingRules()
 
+	vmParams, err := models.NewVictoriaMetricsParams(prometheus.BasePrometheusConfigPath)
```

Comment:
Based on the code looks like we don't need to initialize it here `VM_ENABLED` env var can be extracted in each place separately.

Reply:
So, as far as I see, the only parameter is used in both places is `VM_ENABLED` and it's going to be removed oce we replace Prometheus with Victoria Metrics. At that moment it won't make sense to pass this structure to VM Alert.

Reply:
Let's keep it for now.